shell/components/formatter/WorkloadDetailEndpoints.vue

```diff
@@ -90,7 +75,7 @@ export default {
         :href="endpoint.link"
         target="_blank"
         rel="nofollow noopener noreferrer"
-      ><span v-if="endpoint.port">{{ endpoint.port }}/</span>{{ endpoint.protocol }}</a>
+      >{{ endpoint.link }}</a>
```

Comment:
This changes the display for the service case as well - do we want to do this?

I'd suggest leaving it as is - can we add a property to the endpoint that is the what should be displayed for the link and set that in the loop - the services can use the original from the template.

Reply:
I think I interpreted this correctly @nwmac 🤞 https://github.com/rancher/dashboard/pull/10380/commits/9defbd34b1f1863fa746dc12f81079d2d07321ac

PR is ready for re-review